src/frontend/src/view/dashboard/dashboard.tsx

```diff
@@ -73,7 +76,16 @@ const mapStateToProps = (state: AppState): StateProps => ({
     isLoadingData: selectIsLoadingDeploys(state),
     applications: selectApplicationsWithChanges(state),
     deploys: selectDeploys(state),
+    valgtTeam: getValgtTeam(state),
     showAll: state.view.showAll
 });
 
-export default connect(mapStateToProps)(Dashboard);
+function mapDispatchToProps(dispatch: Dispatch<Action>): DispatchProps {
+    return {
+        getDeploys: (teamId) => {
+            dispatch(getAllDeploys(teamId));
+        }
+    };
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(Dashboard);
```

Comment:
Denne komponenten ser ut til å plutselig få ansvar for å hente data, noen spesiell grun til deploys ikke kan hentes i initial-data-loader?